3rdparty/bx/include/bx/platform.h

```diff
@@ -133,8 +133,14 @@
 #endif //
 
 #if BX_CPU_PPC
-#	undef  BX_CPU_ENDIAN_BIG
-#	define BX_CPU_ENDIAN_BIG 1
+// _LITTLE_ENDIAN exists on ppc64le.
+#	if _LITTLE_ENDIAN
+#		undef  BX_CPU_ENDIAN_LITTLE
+#		define BX_CPU_ENDIAN_LITTLE 1
+#	else
+#		undef  BX_CPU_ENDIAN_BIG
+#		define BX_CPU_ENDIAN_BIG 1
+#	endif
```

Comment:
Have you submitted this upstream ([bkaradzic/bx](https://github.com/bkaradzic/bx))?  We prefer not to get out-of-sync.

Reply:
No. I can do so, but how often do you pull these changes? Do you do this automatically?

Reply:
We do it periodically, or when it becomes necessary for compatibility with things.

Reply:
How do I get that to occur? I could try to do it myself with another PR when I get it done upstream, I guess, or I could file an issue when it's ready to be integrated. The reason I'm a little concerned about the timing here is that the compile will still be broken on ppc64le until that happens.

Reply:
bgfx usually integrates pretty cleanly. Just sling us an issue when it's updated upstream, and tag me in the filing, and I'll see about getting it merged in. Given the holidays and how close the upcoming code freeze is, it's unlikely to happen in time for 0.205, but 0.206 is a distinct possibility.